.github/workflows/integration.yml

```diff
@@ -0,0 +1,20 @@
+name: Cypress Tests
+on: [push]
+jobs:
+  cypress-run:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2
+      - name: Install node
+        uses: actions/setup-node@v2
+        with:
+          node:version: "16"
+      - name: Install dependencies
+        run: npm install
+      - name: Build
+        run: npm run dist
```

Comment:
do you need to build storybook instead of the `dist` folder?

Reply:
It has thia before the integrations tests in the jenkinsfile 
```
      stage('Build') {
          steps {
            ansiColor('xterm') {
              sh '''npm run dist'''
            }
          }

          post {
            always {
              archiveArtifacts 'dist/**/*'
            }
          }
        }
        ```
I wrestled with this test for quite a while and it didn't work unless I had this build action with npm run dist. The `test-integration` file for the `npm run test:integration` takes care of running storybook and then a `npx cypress run`.  Open to suggestions if you have any ideas!

Reply:
I wrestled with this test for quite a while and it didn't work unless I had this build action with npm run dist. The `test-integration` file for the `npm run test:integration` takes care of running storybook and then a `npx cypress run`.  Open to suggestions if you have any ideas!

Reply:
no it must rely on it under the hood. Sorry for the misdirection